### PR TITLE
Change link numbers for avaliable cities

### PR DESCRIPTION
**Pull request by hallaathrad**

### CODE REVIEW

File: List-of-Free-Code-Camp-city-based-Campsites.md [hunk 1]
```diff
@@ -662,7 +662,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
 - South Korea
   - [Busan](https://www.facebook.com/groups/free.code.camp.busan/)
   - [Daegu](https://www.facebook.com/groups/free.code.camp.your.daegu/)
-  - [Gwangju](https://www.facebook.com/groups/204942919854198/)
+  - [Gwangju](https://www.facebook.com/groups/GwangjuCodeCano/)
   - [Jeonju](https://www.facebook.com/groups/free.code.camp.jeonju/)
```
Comment (SaintPeter) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: We should encorage them to follow the naming convention of `free.code.camp.Gwengju.Code.Cano` or whatever that translates to.

File: List-of-Free-Code-Camp-city-based-Campsites.md [hunk 2]
```diff
@@ -851,7 +852,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
     - [Columbus](https://www.facebook.com/groups/free.code.camp.columbus.georgia/)
     - [Macon](https://www.facebook.com/groups/703656036438040)
     - [Savannah](https://www.facebook.com/groups/free.code.camp.savannah/)
-    - [South Atlanta suburbs](https://www.facebook.com/groups/933248003379232/)
+    - [South Atlanta suburbs](https://www.facebook.com/groups/free.code.camp.south.atlanta.suburbs/)
```
Comment (SaintPeter) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: Title Case?  "South Atlanta Suburbs"

File: List-of-Free-Code-Camp-city-based-Campsites.md [hunk 3]
```diff
@@ -1051,7 +1052,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
     - [Memphis](https://www.facebook.com/groups/free.code.camp.Memphis/)
     - [Nashville](https://www.facebook.com/groups/free.code.camp.nashville/)
   - Texas
-    - [Amarillo](https://www.facebook.com/groups/1553257601654900/)
+    - [Amarillo](https://www.facebook.com/groups/freecodecampamarillotexas/)
```
Comment (SaintPeter) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: See prior note about naming convention.

File: List-of-Free-Code-Camp-city-based-Campsites.md (hunk 1 above)
Comment (hallaathrad) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: sure it was m>n and p>o mistyped.

Sadly, sometimes I just receive their already defined link (which we cannot possibly physically enforce for them to get right before they do it themselves), it's already in a different convention —in spite of having clear suggestions in the guide...

And then FB in their infinite wisdom decided they won't let groups change the link once it's been defined. Between a rock and a hard place on this one.

**Pull request merged**